lib/rwssmg.gi

```diff
@@ -36,11 +36,11 @@ function(kbrws)
   SetIsConfluent(rws,true);
   SetFamilyForRewritingSystem(rws, FamilyForRewritingSystem(kbrws));
   SetOrderingOfRewritingSystem(rws,OrderingOfRewritingSystem(kbrws));
-	if IsElementOfFpSemigroupFamily(FamilyForRewritingSystem(kbrws)) then
-		SetIsBuiltFromSemigroup(rws,true);
-	elif IsElementOfFpMonoidFamily(FamilyForRewritingSystem(kbrws)) then
-		SetIsBuiltFromMonoid(rws,true);
-	fi;
+        if IsElementOfFpSemigroupFamily(FamilyForRewritingSystem(kbrws)) then
```

Comment:
I think this needs 2-wide tabs (possibly the whole file)

Reply:
Fixed